### DISCUSSION TITLE
v8.0.0-beta.1: removing a host fails with database is locked (5) (SQLITE_BUSY)

### DISCUSSION THREAD

**Discussion by lovekai47** [ccfos/nightingale#2363] [Q&A]
二进制部署，数据库mysql。移除机器时保存。请大佬们帮忙解决
![image](https://github.com/user-attachments/assets/2332dcbb-6d45-420b-b5dc-a20cceeaeae2)

**Comment by UlricQin** [comment]
在群里跟你说了不是，`show processlist` 看看哪个 sql 在跑，kill 掉试试

**Reply by yeaze** [reply]
看他的报错，用的应该是sqlite， sqlite没有 `show processlist;` 命令的

**Comment by lovekai47** [comment]
sql列表里并没有其他进程
MariaDB [n9e_v6]> SHOW PROCESSLIST;
+----+------+-----------+--------+---------+------+----------+------------------+----------+
| Id | User | Host      | db     | Command | Time | State    | Info             | Progress |
+----+------+-----------+--------+---------+------+----------+------------------+----------+
| 50 | root | localhost | n9e_v6 | Query   |    0 | starting | SHOW PROCESSLIST |    0.000 |
+----+------+-----------+--------+---------+------+----------+------------------+----------+
1 row in set (0.000 sec)
